contentcuration/contentcuration/templates/policies/policy_accept.html

```diff
@@ -22,11 +22,16 @@ <h4>{% trans 'We have updated our policies. Please accept the following terms to
 	</form>
 </div>
 <script>
-	window.addEventListener('load', function(){
-		$("#id_accepted").on("change", function(event){
-			(event.target.checked)? $("#submit").removeAttr("disabled").removeClass("disabled") :
-				$("#submit").attr("disabled", true).addClass("disabled");
-		});
-	});
+	var submitBtn = document.getElementById("submit")
+
```

Comment:
jQuery was injected into the page thanks to `base.js` that has been removed from our bundles (https://github.com/learningequality/studio/pull/1487). But even when I returned it, in some situations, jQuery wasn't found fast enough. So I just got rid of this dependency here for now. I suppose we'll refactor those pages to Vue anyway so I don't need to spend time on getting $ back, is that right?